### ISSUE TITLE
Add support for Postgres COMMENT ON to generate JS docstring

### ISSUE THREAD

**Issue by rattrayalex** [rmp135/sql-ts#68]
Docs: https://www.postgresql.org/docs/current/sql-comment.html

Given this table:
```sql
create table foo (
  bar text
)
comment on table foo is 'Table Foo has everything.
Even bars.'
comment on column foo.bar is 'The bar is closed.'
```
I would like this generated:
```ts
/**
 * Table Foo has everything.
 * Even bars.
 */
export interface FooEntity {
  /** The bar is closed. */
  bar: string
}
```

I believe that https://github.com/kristiandupont/kanel supports this. 

Actually, looking at their [examples](https://github.com/kristiandupont/kanel/blob/master/example/models/Film.ts), It would also be awesome to print comments with more information about the field (like index, default value, etc). This way when you are in your application code in your IDE, you can easily see whether a column has an index, for example.

**Comment by rmp135** [comment]
Should be possible. The templating system allows anything to be put into a comment as long as it's available, we just have to extract it for use. 

I'll take a look into pulling out Postgres comments and other supplemental info that might be useful as well.

**Comment by rattrayalex** [comment]
OK cool! This discussion may also be relevant, maybe it will inspire further ideas: https://github.com/kristiandupont/kanel/issues/127

**Comment by rmp135** [comment]
This is great, thanks for the comments and help with getting this working. I always like to implement the same feature across all clients if possible. SQL Server has extended properties and MySQL has comments, SQLite has no comparison so we can leave that one out. Do need to change the queries up to look in different places but should still be possible to extend.

Regarding adding more verbose comments to the tables / columns. I did think of that but was of two minds about what and how much to add. For now I'll look into adding a helper function to the template compiler to add all the other info we store (original table name, is primary key and so on) as a comment that can be added to various elements if it's wanted.

**Comment by rmp135** [comment]
I've implemented importing comments from tables, views and columns based on the following. I've tested across all providers and they work as intended far as I can tell. It's quite nice as it adds the comment to the TypeScript popover.

Provider   | Source
-----------|---------------------------------------------
MySQL      | "Comments" field
SQL Server | Extended property with name "MS_Description"
Postgres   | "Comment" field
SQLite     | Not supported

I haven't implemented the more verbose commenting. Most of the info can already be added using the template and hopefully this comment extraction can help with anything more expressive.
